pkg/chart/chart.go

```diff
@@ -450,7 +450,6 @@ func (c *ChartClientFactory) New(repoType, userAgent string) ChartClient {
 	switch repoType {
 	case "oci":
 		client = NewOCIClient(userAgent)
-		break
```

Comment:
Interesting, didn't know that Go switch statements do not fall through by default.